test-files/cmd/glacial/ServiceOsmApiDbDcConflateTest.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+test-files/cmd/glacial/OsmApiDbConflate.off test-files/DcGisRoads.osm test-files/DcTigerRoads.osm -77.040,38.892,-77.033,38.896 ServiceOsmApiDbDcConflateTest
```

Comment:
I don't think I do.  I went with that up front, b/c I was comfortable with seeing that dataset conflated.  I'll look into replacing the running test with the AllDataTypes data.  Since the main script just takes in two datasets as inputs, its easy to spin up the same test with DC streets really quickly if I want to see that scenario again in the future for debugging purposes.